src/plugins/condition/plugin.js

```diff
@@ -54,7 +54,12 @@ export default function ConditionPlugin() {
                 domainObject.telemetry = {};
             }
         });
-        openmct.composition.addPolicy(new ConditionSetCompositionPolicy(openmct).allow);
+        openmct.legacyExtension('policies', {
```

Comment:
This might be a merge error, this has been removed in master.

Reply:
Agreed, it was a merge error. I've pushed an update to remove the legacyExtension.